package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suggestions",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Per semver, I think this would be a minor release with the addition of disabling hiding on blur? So `1.7.0`.